src/profile/external-identities/ExternalIdentities.tsx

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash/fp';
+import React from 'react';
+import { PageBox, PageBoxVariants } from 'src/components/PageBox';
+import { getConfig } from 'src/libs/config';
+import allProviders from 'src/libs/providers';
+import { FenceAccount } from 'src/profile/external-identities/FenceAccount';
+import { NihAccount } from 'src/profile/external-identities/NihAccount';
+import { OAuth2Link } from 'src/profile/external-identities/OAuth2Link';
+import { oauth2Provider } from 'src/profile/external-identities/OAuth2Providers';
+
+type ExternalIdentitiesProps = {
+  queryParams: { [key: string]: string };
+};
+
+export const ExternalIdentities = (props: ExternalIdentitiesProps) => {
```

Comment:
Nit: convention for React components.

```suggestion
export const ExternalIdentities = (props: ExternalIdentitiesProps): ReactNode => {
```